test/user_test.go

```diff
@@ -0,0 +1,89 @@
+package test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/NebulousLabs/skynet-accounts/database"
+
+	"gitlab.com/NebulousLabs/fastrand"
+)
+
+func TestUserBandwidth(t *testing.T) {
+	ctx := context.Background()
+	db, err := database.New(ctx, DBTestCredentials(), nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Add a test user.
+	sub := string(fastrand.Bytes(userSubLen))
+	u, err := db.UserCreate(nil, sub, database.TierPremium5)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer func(user *database.User) {
+		_ = db.UserDelete(nil, user)
+	}(u)
+
+	// skylinkLen is the upload's file size, not the used storage.
```

Comment:
I don't see `skylinkLen` anywhere

Reply:
Forgotten comment that applied to a now-removed piece of code. Thanks for spotting it.